### ISSUE TITLE
I have try to use IAPHelper,but it always present cannot connect to iTunes Store 0,Can you tell me why?

### ISSUE THREAD

**Issue by bianxiang** [saturngod/IAPHelper#31]
I have try to use IAPHelper,it can show goods information,but it always present cannot connect to iTunes Store 0,Can you tell me why?
![047fb9b8-4408-414d-8298-3c46ee8e07b3](https://cloud.githubusercontent.com/assets/16380379/16480480/0a74057c-3ed9-11e6-90f5-f0734b6b6715.png)

**Comment by CeccoCQ** [comment]
Same problem here, but only in production mode.
In sandbox all works fine.

**Comment by saturngod** [comment]
@bianxiang 
Hi , can you logout your app store and login with test account if you are not on production mode.

@CeccoCQ can you try to logout the app store account and try again in production mode ? It happen sometime if we was testing with test account.

**Comment by CeccoCQ** [comment]
@saturngod I've already tried to logout, without success. In sandbox mode all works correctly.

**Comment by SoaringDovahkiin** [comment]
Yes, I got the same problem as well. Very disappointing as I switched from MKStoreKit to this in the hopes that this might work; problem might be related to IPv6 networks.

**Comment by saturngod** [comment]
@SoaringDovahkiin happen in production ? How did you test the production ? If it's Adhoc version cannot use with real apple store account. Need to use with test account.

Real app store account can only use for download from app store.

**Comment by SoaringDovahkiin** [comment]
@saturngod Yes, it was on the iPhone 6 simulator on a sandbox environment (test account) and got the following error: Fail Cannot connect to iTunes Store. I will be testing this out on a real device as well.

**Comment by jonkrider** [comment]
Has anyone solved this problem yet? I am have the exact same issue, and have been working on it for over a week now. I like IAPHelper and would like to use it.

**Comment by yuyangdexue** [comment]
why

**Comment by yuyangdexue** [comment]
@jonkrider  why ？ it always present cannot connect to iTunes Store 0

**Comment by poongd** [comment]
same issue

**Comment by xyxdasnjss** [comment]
猜测可能是因为你的 AppleID 是正式的，需要 用测试账号 进行购买测试；
注销掉AppleID，然后运行购买代码，这时应该会 询问你的 测试账号。

**Comment by revolter** [comment]
@xyxdasnjss, Maybe use english so it may help others having the same problem?
